### ISSUE TITLE
FAILURE: Build failed with an exception due to miss “org/apache/commons/io/IOUtils”

### ISSUE THREAD

**Issue by TonyTangAndroid** [KeepSafe/dexcount-gradle-plugin#2]
Hi there, I am really excited to know about this gradle plugin as it definitely comes in handy. However, this is what I have encounter:

>*#FAILURE: Build failed with an exception.

>What went wrong:
>Execution failed for task ':app:assembleOfficialDebug'.
> org/apache/commons/io/IOUtils*

**Comment by benjamin-bader** [comment]
Thanks for the report!  This should be fixed as of 828a732 (i.e. five minutes ago); next time you try, please let me know if it works.

**Comment by TonyTangAndroid** [comment]
Thank you very much for such quick response. I had just tried it, now it works good. 

Just for those who still had run into the issue, you need to clear the cache related to dexcount-gradle-plugin in your .gradle folder. By simply invalidating and restart Android Studio won't solve this problem if you already downloaded the mis-configured version before the fix has come. 

I suggest the best way to avoid such manually clearance of dexcount-gradle-plugin cache is to update the version number from 

 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.1.0-SNAPSHOT'

to 

 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.1.1-SNAPSHOT'.

It will be appreicated if anyone knows a better way to clear the gradle plugin cache in Android Studio.

**Comment by benjamin-bader** [comment]
Hm, that sounds wrong - the whole point of snapshot dependencies is that they are updated on every build!

Glad to hear it's working again.

**Comment by TonyTangAndroid** [comment]
Thank you very much for your reminder of the purpose of Snapshot. I did not realize that. And I had felt curious that why I still had to go through all that trouble to get it working. And I found answer [here](http://stackoverflow.com/a/5907727/4068957), which states:

*However, this check is made only once per day. That means that if you have a foo-1.0-20110506.110000-1.jar (i.e. this library has been generated on 2011/05/06 at 11:00:00) in your local repository, and if you run the Maven build again the same day, Maven will not check the repositories for a newer version.*

Thank you again for your feedback and hardwork.
